### ISSUE TITLE
Затирание директории

### ISSUE THREAD

**Issue by Daniilisi** [abbat/ydcmd#30]
текущее состояние:
disk:/data/documents/1.txt
disk:/data/documents/1.txt
disk:/data/documents/3.txt

При использовании команды put
```ydcmd put file.dat data```
затираются все файлы из директории без возможности восстановить.
Добавьте пожалуйста подтверждение при попытке затереть директорию с файлами.
Затер сегодня таким образом кучу важных документов забыв добавить в конце / =(

**Comment by abbat** [comment]
См. #23
Для удаления в корзину во избежании несчастных случаев рекомендуется использовать параметр `--trash`

**Comment by Daniilisi** [comment]
Так тут вопрос не удаления, а загрузки и возможной ошибки при указании пути. В этом случае происходит затирание без возможности восстановить, может стоит подтверждение добавить при таких сценариях?

**Comment by abbat** [comment]
> затирание без возможности восстановить

Теоретически как раз параметр --trash позволит восстановить перезатертую директорию в случае ошибки. Корзина сама очищается через какое-то количество дней или можно ее очистить этой же утилитой.

> может стоит подтверждение добавить при таких сценариях

Основной сценарий использования утилиты - скрпты автоматического резервного копирования. В автоматических сценариях, запускаемых по cron, если она будет ждать ввода пользователя, то ей никто не сможет ответить. Ломать это поведение сейчас уже нельзя, но можно попробовать сделать через какой-нибудь параметр.

**Comment by Daniilisi** [comment]
Да, точно, если встроить это возникнут проблемы у пользователей, которые уже настроили автоматизацию. (как-то я не подумал, извиняюсь).
Если можно тогда, ключик какой-нибудь или параметры в конфиг файл, для этого.
Мне кажется вы недооцениваете, использование вашего скрипта, он вполне удобен и для хранения файлов и управления как самостоятельный консольный клиент, большое спасибо за это!
